src/techs/techs.controller.ts

```diff
@@ -181,4 +183,45 @@ export class TechsController {
     ) {
         return this.techsService.removeVote(req, teamId, teamTechId);
     }
+
+    @ApiOperation({
+        summary:
+            "Updates arrays of tech stack items, grouped by categoryId, sets 'isSelected' values",
+        description:
+            "Maximum of 3 selections per category allowed. Login required",
```

Comment:
```suggestion
            "Maximum of 3 selections per category allowed. Login required. All tech items (isSelected === true/false) are required",
```

Reply:
That is a good catch.  Yes, it is assumed that all techs in a category will be submitted together in the same array.  It counts all `true` values from this array, and does not check the DB.  Also - the `categoryId` field is not used and is only there for readability at this point.